File: TP2/Python/Punto3/Alumno.py
class Alumno:
    #def __init__(self):
    # __nombre =''
    #__apellido = ''
    #__dni = 0
    
    def __init__(self, nombre='', apellido='', dni=0):
        self.__nombre = nombre
        self.__apellido = apellido
        self.__dni = dni
    
    #def __init__(self):
    #    pass
    
    @classmethod
    def iniciar_con_nom_ape(cls, nombre, apellido):
        alumno = cls.__new__(cls)
        alumno.__nombre = nombre
        alumno.__apellido = apellido
        return alumno
    
    @property
    def nombre(self):
        return self.__nombre
    
    @nombre.setter 
    def nombre(self, valor):
        self.__nombre = valor
        
    @property
    def apellido(self):
        return self.__apellido
    
    @apellido.setter 
    def apellido(self, valor):
        self.__apellido = valor
    
    @property
    def dni(self):
        return self.__dni
    
    @dni.setter 
    def dni(self, valor):
        self.__dni = valor
    
    def getNombreYapellido(self):
        return f"{self.__nombre} {self.__apellido}"

File: TP2/Java/Punto 3/Alumno.java

public class Alumno {
    private String nombre = "";
    private String apellido = "";
    private int dni = 0;

    public Alumno(String nombre, String apellido, int dni) {
        this.nombre = nombre;
        this.apellido = apellido;
        this.dni = dni;
    }
    public Alumno(){
        this.nombre = "";
        this.apellido = "";
        this.dni = 0;
    }
    public Alumno(String nombre, String apellido){
        this.nombre = nombre;
        this.apellido = apellido;
        this.dni = 0;
    }
    public void imprimir(){
        System.out.println("Nombre: " + this.nombre);
        System.out.println("Apellido: " + this.apellido);
        System.out.println("DNI: " + this.dni);
    }
    public void setNombre(String nombre){
        this.nombre = nombre;
    }
    public String getNombre(){
        return this.nombre;
    }
    public void setApellido(String apellido){
        this.apellido = apellido;
    }
    public String getApellido(){
        return this.apellido;
    }
    public void setDni(int dni){
        this.dni = dni;
    }
    public int getDni(){
        return this.dni;
    }
    public String getNombreYapellido(){
        return this.nombre + " " + this.apellido;
    }
    
}
